start_pokealarm.py

```diff
@@ -195,12 +209,15 @@ def check_for_update():
     for file_ in tmp_files:
         os.remove(file_)
 
-    if (not os.path.isfile('data/pokemon_data.json') or
-        not os.path.isfile('data/fast_moves.json') or
-            not os.path.isfile('data/charged_moves.json') or
-            not os.path.isfile('data/shiny_data.json') or
-            not os.path.isfile('data/invasions.json')):
-        log.critical("Missing PokeAlarm data")
+    missing_data_detected = False
+    for dataname in ['pokemon_data', 'fast_moves', 'charged_moves',
+                     'shiny_data', 'invasions', 'rankings-1500',
+                     'rankings-2500', 'rankings-10000']:
+        if not os.path.isfile(f'data/{dataname}.json'):
+            log.critical(f"Missing PokeAlarm data: {dataname}")
```

Comment:
maybe be a bit more explicit with this log
`Missing required external PokeAlarm data:`
This way people don't think it's missing internal files

Reply:
I agree